src/main/drivers/pwm_output.h

```diff
@@ -54,6 +54,9 @@ extern FAST_DATA_ZERO_INIT pwmOutputPort_t motors[MAX_SUPPORTED_MOTORS];
 extern FAST_DATA_ZERO_INIT uint8_t pwmMotorCount;
 
 bool motorPwmDevInit(motorDevice_t *device, const motorDevConfig_t *motorDevConfig, uint16_t idlePulse);
+IO_t pwmGetMotorIO(unsigned index);
```

Comment:
(ideally this would go into pwm_output_impl.h - it is implementation detail that should not be part of public pwm interface; old functions were `static`)